package/android/gradle.properties

```diff
@@ -1,4 +1,4 @@
-ReactNativeSlider_compileSdkVersion=29
+ReactNativeSlider_compileSdkVersion=31
 ReactNativeSlider_buildToolsVersion=28.0.3
-ReactNativeSlider_targetSdkVersion=27
-ReactNativeSlider_minSdkVersion=16
\ No newline at end of file
+ReactNativeSlider_targetSdkVersion=31
+ReactNativeSlider_minSdkVersion=21
```

Comment:
I'm worried about the backward compatibility here,
while at the same time it feels odd leaving those at such low numbers.
Maybe we should keep the minimum intact, but update the target?